src/FedKeyPairGen/Program.cs

```diff
@@ -167,10 +176,12 @@ private static void GeneratePublicPrivateKeys()
             tool.SavePrivateKey(key);
             PubKey miningPubKey = key.PubKey;
 
+            var outputPassphrase = passphrase == null ? string.Empty : $" | passphrase={passphrase}";
+
             Console.WriteLine($"-----------------------------------------------------------------------------");
             Console.WriteLine($"-- Please give the following 2 public keys to the federation administrator --");
             Console.WriteLine($"-----------------------------------------------------------------------------");
-            Console.WriteLine($"1. Your signing pubkey: {Encoders.Hex.EncodeData(signingPubKey.ToBytes(false))}");
+            Console.WriteLine($"1. Your signing pubkey: {Encoders.Hex.EncodeData(signingPubKey.ToBytes(false))}{outputPassphrase}");

```

Comment:
This should probably be under `Your signing mnemonic:` rather than under Give this to your admin :-)

Reply:
that should be better, sorry about that